### PR TITLE
[WIP] Provide the service enviornment information

### PR DESCRIPTION
**Pull request by crodjer**
In order for the clients to identify which service they are dealing with, provide the running environment in json responses. Being aware of the environment will help in debugging issues with applications.

This patch may need more tests. I tested it out with simplepush_test locally and added a basic test case.

Ref: #50

### CODE REVIEW

File: autopush/websocket.py [hunk 1]
```diff
@@ -394,6 +394,8 @@ def returnError(self, messageType, reason, statusCode, close=True):
 
     def sendJSON(self, body):
         """Send a Python dict as a JSON string in a websocket message"""
+
+        body['env'] = self.ap_settings.env
```
Comment (bbangert) [inline autopush/websocket.py]: I think it should only go in the hello message. Not every single message.

**Comment by bbangert** [comment]
If you could rebase to the latest, and update, it looks good. Still needs a changelog entry for it.

**Comment by crodjer** [comment]
@bbangert Thanks for the review, I'll incorporate your changes in the PR.

**Comment by crodjer** [comment]
Updated the pull request.

File: autopush/settings.py [hunk 2]
```diff
@@ -151,6 +152,10 @@ def __init__(self,
         if 'gcm' in router_conf:
             self.routers["gcm"] = GCMRouter(self, router_conf["gcm"])
 
+
```
Comment (jrconlin) [inline autopush/settings.py]: Bummer. Need to remove this extra linespace and then all tests will pass. (The test is barking because of Flake8.)

**Comment by crodjer** [comment]
@jrconlin Thanks for pointing that out. Fixed.

**Comment by jrconlin** [comment]
Woot! r+

**Comment by crodjer** [comment]
Thanks

**Pull request merged**